src/mds/aliases.py

```diff
@@ -110,33 +107,32 @@ async def update_metadata_alias(
         guid (str): Metadata GUID
         body (AliasObjInput): JSON body with list of aliases
         request (Request): incoming request
-        merge (bool, optional): Description
+        merge (bool, optional): If `merge` is True, then any aliases that are not
+            in the new data will be kept.
     """
+    input_body_aliases = body.aliases or []
+    requested_aliases = set(input_body_aliases)
+    logger.debug(f"requested_aliases: {requested_aliases}")
```

Comment:
remove this debug log?

Reply:
I've been opting to leave in some debug logs I think might aide in future debugging if we need to turn them on. I'm maybe biased because I've been a little annoyed with older code _not_ having many debug logs, so maybe I'm overcompensating here. 

Let me know what you think, I guess I could go either way